ee/query-service/app/server.go

```diff
@@ -199,17 +202,61 @@ func (s *Server) createPrivateServer(apiHandler *api.APIHandler) (*http.Server,
 	}, nil
 }
 
+func getPATToken(r *http.Request) (string, error) {
+	patHeader := r.Header.Get("SIGNOZ-API-KEY")
+	if patHeader == "" {
```

Comment:
Please add a debugf message to indicate if PAT was received. 

This would help in handling support issues around auth.

Reply:
Done